pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/SparkSegmentMetadataPushIntegrationTest.java

```diff
@@ -106,13 +106,13 @@ public void setUp()
     _sparkContext = new SparkContext("local", SparkSegmentMetadataPushIntegrationTest.class.getName());
   }
 
-  @Test
+  @Test(priority = 1)
```

Comment:
Are changes in this file related?

Reply:
These were the tests that were failing when run out of order. This just maintains the ordering in which they were written (and, I suppose, intended to be run)

Reply:
Have you figured out the reason why it is failing? Seems this test is not related to this PR. Consider making this change in a separate dedicate PR

Reply:
I have not. I removed this ordering from this PR, but I noticed that it fails locally when unordered even in a direct mirror of master (without my changes)